README.md

```diff
@@ -63,6 +63,12 @@ ANSIBLE_LOCAL_TMP = '~/.ansible/tmp'
 
 - `LOCAL_IMAGE_DOCKER` - Set to `True` to run the `ansible-test` container image via Docker; otherwise, Podman will be used. Defaults to `False`.
 
+- `RUN_ANSIBLE_LINT_ROLES` - Set to `True` to run `ansible-lint` on roles (only). Defaults to `False`.
+
+- `RUN_ANSIBLE_LINT_COLLECTION` - Set to `True` to run `ansible-lint` on the entire collection. Defaults to `True`.
```

Comment:
Calling ansible-lint for a role inside a collection was a workaround for when there was no ability to run ansible-lint at the collection level. Now we are implementing that feature. Thinking about this more, I’m leaning towards not maintaining that workaround. By linting the entire collection we are still linting the roles, we are just changing how they are called. People will see more lint output, but that does stop a collection from being imported, so we are not breaking compatibility. I think this is preferable to maintaining multiple ways to call and having more complex configuration options.

I propose we run ansible-lint at the collection level when `run_ansible_lint` is True, and remove running at the role-inside-a-collection level, and keep the config option as-is.

Reply:
This is fine by Partner Engineering. So we'd fully remove the existing call in content.py?

Reply:
Refactored to remove the loader level call.